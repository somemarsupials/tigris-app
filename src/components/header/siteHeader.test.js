import React from 'react';
import { shallow } from 'enzyme';
import SiteHeader from './siteHeader';

describe('SiteHeader', () => {
  it('matches snapshot', () => {
    expect(shallow(<SiteHeader />)).toMatchSnapshot();
  });
});
