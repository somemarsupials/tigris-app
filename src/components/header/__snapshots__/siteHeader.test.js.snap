// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteHeader matches snapshot 1`] = `
<div>
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
